### DISCUSSION TITLE
【select】select组件使用options属性替代Option组件时，disabled属性如何设置？

### DISCUSSION THREAD

**Discussion by tianzhich** [ant-design/ant-design#28358] [Q&A] [answered]
文档说使用options属性可以获得更好性能：
![image](https://user-images.githubusercontent.com/33218310/102159550-bb2e1580-3ebe-11eb-921e-90d14ca206ed.png)

而且使用filterOption搜索时，第二个参数回传内容自动包含label而不是option组件实例，因此无需使用option.props.children方式进行label匹配，换成options.children，虽然支持，但是也标上了deprecated标签
![image](https://user-images.githubusercontent.com/33218310/102159567-c5e8aa80-3ebe-11eb-956f-09698037fd35.png)

但是有一个问题就是Option组件可以设置disabled，但是这种方式如何设置呢？

版本：antd@4.9.2

提前感谢！

**Comment by yoyo837** [comment, marked as answer]
```js
options={[{
    disabled: true
}]}
```

试试

**Reply by tianzhich** [reply]
感谢，确实可以设置这个属性

**Comment by yoyo837** [comment]
jsx换js就不习惯啦？😂

**Reply by tianzhich** [reply]
哈哈，当时应该试一下的，我就看了文档，以为只能设置`label`和`value`这俩，文档那儿我建议可以加上...啥的，避免有我这样的憨憨🐶

**Reply by yoyo837** [reply]
可以帮忙PR哦，有时候遗漏，或者不全。
